ioctl/newcmd/account/account.go

```diff
@@ -182,21 +183,24 @@ func PrivateKeyFromSigner(signer, password string) (crypto.PrivateKey, error) {
 }
 
 // GetAccountMeta gets account metadata
-func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
-	conn, err := util.ConnectToEndpoint(config.ReadConfig.SecureConnect && !config.Insecure)
+func GetAccountMeta(addr string, client ioctl.Client) (*iotextypes.AccountMeta, error) {
+	var endpoint string
+	var insecure bool
+	apiServiceClient, err := client.APIServiceClient(ioctl.APIServiceConfig{
+		Endpoint: endpoint,
+		Insecure: insecure,
+	})
 	if err != nil {
-		return nil, output.NewError(output.NetworkError, "failed to connect to endpoint", err)
+		return nil, err
 	}
-	defer conn.Close()
```

Comment:
and no need to close the client?

Reply:
NewClient()  is not implemented by main ,  close client is not add now.  when add NewClient,  add close client.  this can be done by a new pr.  this pr is just define the content of command, but not execute  cmd.

Reply:
ok, so here's my suggestion:
1. for this PR, let's add the `nonce` command into current old ioctl
2. a new PR to fix the "close client is not add now" issue for the new ioctl client
3. finally, we can have a PR to migrate the old account commands to new ioctl client